jsplot/src/main/java/tech/tablesaw/plotly/api/AreaPlot.java

```diff
@@ -35,24 +30,8 @@ public static Figure create(String title, Table table, String xCol, String yCol,
         return new Figure(layout, traces);
     }
 
-    private static Layout.LayoutBuilder standardLayout(String title, String xCol, String yCol) {
-        return Layout.builder()
-                    .title(title)
-                    .height(HEIGHT)
-                    .width(WIDTH)
-                    .showLegend(true)
-                    .xAxis(Axis.builder()
-                            .title(xCol)
-                            .build())
-                    .yAxis(Axis.builder()
-                            .title(yCol)
```

Comment:
is it me or do you lose the default use of the column titles as the x and y axis titles with the changes you made?

Reply:
I don't think so, do we? The new version passes `xCol` and `yCol` into the `builder` method, which sets them